src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -480,8 +480,7 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 
 	private func showQRScreen(testInformation: CoronaTestQRCodeInformation?, isLoading: @escaping (Bool) -> Void) {
 		let testInformationSuccess: (CoronaTestQRCodeInformation) -> Void = { [weak self] testQRCodeInformation in
-			if let oldTest = self?.model.coronaTestService.coronaTest(ofType: testQRCodeInformation.testType),
-			   oldTest.testResult != .expired && oldTest.testResult != .invalid {
+			if self?.model.coronaTestService.coronaTest(ofType: testQRCodeInformation.testType) != nil {
```

Comment:
I see that the discussion in JIRA wasn't finished, please ask Martin Hermes if we should actually always show that screen now. As you can see in the comments, it was implemented that way after talking to him and he hasn't commented on the issue yet.

Reply:
Has this been clarified?

Reply:
From my understanding of the bug ticket we should only stop skipping for `.invalid` tests, but keep skipping for `.expired` tests

Reply:
Additionally it seems that we forgot to implement skipping this screen for outdated antigen tests, would be great if we could fix that here as well